Tests/SonarScanner.MSBuild.Common.UnitTests/EnvScannerPropertiesProviderTest.cs

```diff
@@ -38,20 +37,22 @@ public void ParseValidJson()
             provider.GetAllProperties().First().Value.Should().Be("http://myhost");
         }
 
-        [TestCleanup]
-        public void Cleanup()
-        {
-            Environment.SetEnvironmentVariable("SONARQUBE_SCANNER_PARAMS", null);
-        }
-
         [TestMethod]
         public void ParseInvalidJson()
         {
             var logger = new TestLogger();
-            Environment.SetEnvironmentVariable("SONARQUBE_SCANNER_PARAMS", "trash");
```

Comment:
I've checked that there are no other tests that set environment variables directly and that they use the _EnvironmentVariableScope_ instead (well, there is one test that still directly sets environment variables, but it does it inside a _try-catch_ and cleans them up afterwards).